scripts/st2bootstrap-el8.sh

```diff
@@ -257,7 +257,12 @@ configure_st2_user () {
   # to be generated again.
   if ! sudo test -s ${SYSTEM_HOME}/.ssh/stanley_rsa; then
     # added PEM to enforce PEM ssh key type in EL8 to maintain consistency
-    PEM="-m PEM"
+    if [[ "$RHMAJVER" != '6' ]]; then
```

Comment:
`RHMAJVER` is not available if script is ran directly as `st2bootstrap-el8.sh`. That's an issue.

Reply:
@armab I think I resolved this: https://github.com/StackStorm/st2-packages/pull/637/files